src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -6181,7 +6420,7 @@ class Program : $$
 
             var referencedCode = @"
 [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-public struct Goo
+public class Goo
```

Comment:
Please don't change inputs to existing tests. Instead show what happens to those inputs, and add new tests to cover cases of you think they need coverage. Thanks!

Reply:
> It is clear, that if Goo is a struct then it is not valid in the given position no matter what value of EditorBrowsableState is. 

Then update the test to show that it is not valid.

If yuo then want to test the `class Goo` case, add a new sibling test.  Thanks!